### PR TITLE
test

### PR DESCRIPTION
**Pull request by ferigis**

### CODE REVIEW

File: correct_file.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
-This file is correct,
-No misspelled word here.
\ No newline at end of file
+This file is correct.
+No misspelled word here.
```
Comment (ferigis) [inline correct_file.md]: I found 2 misspelled words in this sentence: 
- wordds
	- *Suggested Replacements:* worlds, words
- twoo
	- *Suggested Replacements:* thoo, woo, too, twos, two

File: correct_file.md [hunk 2]
```diff
@@ -1,2 +1,4 @@
...
+there are twoo misspelled wordds here.
```
Comment (ferigis) [inline correct_file.md]: I found 1 misspelled word in this sentence: 
- byessdfsf

File: correct_file.md [hunk 3]
```diff
@@ -1,2 +1,4 @@
...
+bye! bye, bye, byessdfsf
```
Comment (sheldonci) [inline correct_file.md]: I found 1 misspelled word in this sentence: 
- byessdfsf

File: correct_file.md (hunk 2 above)
Comment (sheldonci) [inline correct_file.md]: I found 2 misspelled words in this sentence: 
- wordds
	- *Suggested Replacements:* worlds, words
- twoo
	- *Suggested Replacements:* thoo, woo, too, twos, two

**Pull request closed without merging**